src/Compiler/Microsoft.AspNetCore.Razor.Language/test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -2325,7 +2325,7 @@ public void InvalidCode_EmptyTransition()
             ] : [
             // (24,36): error CS1525: Invalid expression term ')'
             //             __builder.AddContent(3, 
-            Diagnostic(ErrorCode.ERR_InvalidExprTerm, "").WithArguments(")").WithLocation(24, 36)
+            Diagnostic(ErrorCode.ERR_InvalidExprTerm, "").WithArguments(")").WithLocation(3, 2)
```

Comment:
`(3,2)` location looks like it now maps to the `.razor` rather than `.razor.g.cs`, right? That means the comment

```cs
// (24,36): error CS1525: Invalid expression term ')'
//             __builder.AddContent(3, 
```

is outdated, right? No need to update it, though, that might be a lot of work to do in all cases, I'm just clarifying.

Reply:
Correct. I actually updated them as there were only a couple of instances.